### ISSUE TITLE
bug: Manager doesn't follow system theme settings immediately 

### ISSUE THREAD

**Issue by ILoveOpenSourceApplications** [ReVanced/revanced-manager#1877]
### Bug description

The manager doesn't change the theme to whatever set by the user while staying within the app. Only after a restart does the changes apply.

https://github.com/ReVanced/revanced-manager/assets/117499019/2b1e517e-4611-414e-af9a-62f07d76b70e

### Version of ReVanced Manager and version & name of app you are patching

v.1.20.1

### Installation method

Regular

### ReVanced Manager logs

```shell
Nil
```

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The bug is only related to ReVanced Manager
